integration/sawtooth_integration/docker/test_peer_list.yaml

```diff
@@ -137,7 +158,10 @@ services:
     stop_signal: SIGKILL
 
   rest-api-2:
-    image: sawtooth-rest-api:$ISOLATION_ID
+    build:
+      context: ../../..
```

Comment:
I think $SAWTOOTH_CORE is already set to this path. Can you use it here as well?

Reply:
I'm not sure that approach works since there are a ton of compose files at different depths.